runtime/compiler/control/rossa.h

```diff
@@ -76,6 +76,13 @@ typedef enum {
    compilationSymbolValidationManagerFailure       = 50,
    compilationAOTNoSupportForAOTFailure            = 51,
    compilationAOTValidateTMFailure                 = 52,
+#if defined(JITSERVER_SUPPORT)
+   compilationStreamFailure                        = 53,
+   compilationStreamLostMessage                    = 54,
+   compilationStreamMessageTypeMismatch            = 55,
+   compilationStreamVersionIncompatible            = 56,
+   compilationStreamInterrupted                    = 57,
+#endif /* defined(JITSERVER_SUPPORT) */
```

Comment:
"   /* If new codes are added then add the corresponding names in compilationErrorNames table in rossa.cpp */ "